Makefile.js

```diff
@@ -197,7 +197,8 @@ function generateRuleIndexPage() {
                         name: basename,
                         description: rule.meta.docs.description,
                         recommended: rule.meta.docs.recommended || false,
-                        fixable: !!rule.meta.fixable
+                        fixable: !!rule.meta.fixable,
```

Comment:
FYI: This is changing in v8.0.0. It will be meta.hasSuggestions. See https://github.com/eslint/eslint/issues/13398

Reply:
I think you meant to link to #14573? I'll update that PR after this is merged to ensure this code refers to `meta.hasSuggestions` once that change takes place.

Reply:
That works too. I was intending to point out that meta.docs.suggestions would be going away.